src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -173,40 +186,68 @@ public async Task QueryTypeColumnFilterAndOrderBy(string type, string filterOper
+            DisplayName = "datetimeoffset type filter and orderby test with eq operator")]
+        [DataRow(DATE_TYPE, "eq", "'1999-01-08'", "\"1999-01-08\"", "=",
+            DisplayName = "date type filter and orderby test with eq operator")]
+        [DataRow(DATE_TYPE, "gte", "'1999-01-08'", "\"1999-01-08\"", ">=",
+            DisplayName = "date type filter test and orderby  with gte operator")]
+        [DataRow(DATE_TYPE, "neq", "'9998-12-31'", "\"9998-12-31\"", "!=",
+            DisplayName = "date type filter test and orderby  with ne operator")]
+        [DataRow(SMALLDATETIME_TYPE, "eq", "'1999-01-08 10:24:00'", "\"1999-01-08 10:24:00\"", "=",
+            DisplayName = "smalldatetime type filter and orderby test with eq operator")]
+        [DataRow(SMALLDATETIME_TYPE, "gte", "'1999-01-08 10:24:00'", "\"1999-01-08 10:24:00\"", ">=",
+            DisplayName = "smalldatetime type filter and orderby test with gte operator")]
+        [DataRow(SMALLDATETIME_TYPE, "neq", "'1999-01-08 10:24:00'", "\"1999-01-08 10:24:00\"", "!=",
+            DisplayName = "smalldatetime type filter and orderby test with neq operator")]
+        [DataRow(DATETIME2_TYPE, "eq", "'1999-01-08 10:23:00.9999999'", "\"1999-01-08 10:23:00.9999999\"", "=",
+            DisplayName = "datetime2 type filter and orderby test with eq operator")]
+        [DataRow(DATETIME2_TYPE, "gt", "'0001-01-08 10:23:00.9999999'", "\"0001-01-08 10:23:00.9999999\"", " > ",
+            DisplayName = "datetime2 type filter and orderby test with gt operator")]
+        [DataRow(DATETIME2_TYPE, "gte", "'0001-01-08 10:23:00.9999999'", "\"0001-01-08 10:23:00.9999999\"", " >= ",
+            DisplayName = "datetime2 type filter and orderby test with gte operator")]
+        [DataRow(DATETIME2_TYPE, "lt", "'0002-06-06'", "\"0002-06-06\"", " < ",
+            DisplayName = "datetime2 type filter and orderby test with lt operator")]
+        [DataRow(DATETIME2_TYPE, "lte", "'9999-12-31'", "\"9999-12-31\"", " <= ",
+            DisplayName = "datetime2 type filter and orderby test with lte operator")]
+        [DataRow(DATETIME2_TYPE, "neq", "'9999-12-31 23:59:59'", "\"9999-12-31 23:59:59\"", "!=",
+            DisplayName = "datetime2 type filter and orderby test with neq operator")]
         public async Task QueryTypeColumnFilterAndOrderByDateTime(string type, string filterOperator, string sqlValue, string gqlValue, string queryOperator)
         {
             // In MySQL, the DATETIME data type supports a range from '1000-01-01 00:00:00.0000000' to '9999-12-31 23:59:59.0000000'
-            if (DatabaseEngine is TestCategory.MYSQL && sqlValue is "\'9999-12-31 23:59:59.9999999\'")
+            if (DatabaseEngine is TestCategory.MYSQL && sqlValue is "'9999-12-31 23:59:59.9999999'")
             {
-                sqlValue = "\'9999-12-31 23:59:59.0000000\'";
+                sqlValue = "'9999-12-31 23:59:59.0000000'";
```

Comment:
What change allowed you to removed the `\` escape? was it just unnecessary to begin with?

Reply:
Yeah, was unnecessary.